src/Components/Catalog/Header/Header.js

```diff
@@ -10,16 +23,31 @@ const Header = () => {
         <header className={style.header}>
             <h1 className={style.logo}>Movies</h1>
             <nav className={style.menu}>
-                {
+                <div className={style.links}>
+                    {
                     mainUrl ? 
-                    <div>
-                        <Link to = '/search'>Search</Link>
-                        <Link to = '/create'>Create</Link>
-                    </div>
-                     : 
-                    <Link to = '/'>Main</Link>
+                            <div>
+                                <Link to = '/search'>Search</Link>
+                                <Link to = '/create'>Create</Link>
+                            </div>
+                        : 
+                            <Link to = '/'>Main</Link>
+                    }
+                </div>
+                {
+                    isActive ? 
+                        <button className={style.adaptive_menu} onClick = {isActiveHandle}>
+                            <CloseAdaptiveMenu/>
+                        </button>
+                    :
+                        <button className={style.adaptive_menu} onClick = {isActiveHandle}>
+                            <AdaptiveMenu/>
+                        </button>
```

Comment:
Write in this way: `<button> { isActive ? closeAdaptiveView : AdaptiveMenu } </button>` 
Because you have the same values for styles and hadler onClick, it makes no sense to duplicate button